pykg2tbl/j2/templates/broader-terms.sparql

```diff
@@ -0,0 +1,18 @@
+{# Jinja Template --> Sparql Query
+ | Get any broader broader terms in a triple.
+ | variables in template:
+ |    - triple: triple to search from
+ |    - (optional) language: language of the collection
+#}
+
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix dc:   <http://purl.org/dc/terms/> 
+SELECT ?bterm ?blbl WHERE {
+  bind ( {{triple}} as ?term) 
```

Comment:
have to rename here too + introduce the <> so the end-user should not provide those

so rather :
`bind ( <{{term}}> as ?term)`